packages/react-components/package.json

```diff
@@ -39,6 +39,7 @@
     "@fluentui/react-window-provider": "~2.2.3",
     "@fluentui/react": "~8.98.3",
     "@internal/acs-ui-common": "1.5.1-beta.2",
+    "@internal/northstar-wrapper": "^0.1.0",
```

Comment:
The version here should match all the other packages under packages.json (yes, all the packages are using the same version number even they are not public)